apps.toml

```diff
@@ -1868,9 +1868,10 @@ url = "https://github.com/YunoHost-Apps/grist_ynh"
 [grocy]
 added_date = 1674232499 # 2023/01/20
 branch = "master"
-category = "small_utilities"
+category = "productivity_and_management"
 level = 8
 state = "working"
+subtags = [ "business_and_ngos" ]
```

Comment:
hmmm not sure about this one ? I think this subtag is more like ERP/business/acconting but this is a fridge manager ? :thinking:

Reply:
i think we should better create a new "home organization" category with a "kitchen" subcategory